app/screens/home/search/index.tsx

```diff
@@ -16,9 +16,12 @@ const enhance = withObservables([], ({database}: WithDatabaseArgs) => {
     return {
         teamId: currentTeamId,
         teams: queryJoinedTeams(database).observe(),
+        crossTeamSearchEnabled: observeConfigBooleanValue(database, 'FeatureFlagExperimentalCrossTeamSearch'),
     };
 });
 
+export const ALL_TEAMS_ID = '';
```

Comment:
perhaps this should be moved to a constants file.

Reply:
I didn't notice the constants folder! Thanks, I moved it there!